client/branded/src/global-styles/colors-redesign.scss

```
@@ -35,9 +35,7 @@ $theme-colors-redesign: (
     merged: $redesign-purple,
 );
 
-:root.theme-redesign,
-// Descendant selector is required for Storybook `addRedesignVariants` helper.
-:root .theme-redesign {
+:root.theme-redesign {
```

Is it important that it's the `<html>` element? 
```suggestion
.theme-redesign {
```

---

That should be fine to change I believe, I think we would need the descendant selector here otherwise as these classes are set on the `<body>` in Storybook